tests/checks/remotesettings/test_collections_consistency.py

```diff
@@ -4,7 +4,9 @@
 
 
 FAKE_AUTH = "Bearer abc"
-COLLECTION_URL = "/buckets/{}/collections/{}"
+COLLECTIONS_URL = "/buckets/{}/collections"
+COLLECTION_URL = COLLECTIONS_URL + "/{}"
+CHANGESET_URL = "/buckets/{}/collections/{}/changeset"
```

Comment:
Maybe
```suggestion
CHANGESET_URL = COLLECTION_URL + "/changeset"
```
to be consistent with the previous and following line?